src/tools/annotation/CircleRoiTool.js

```diff
@@ -121,8 +121,9 @@ export default class CircleRoiTool extends BaseAnnotationTool {
     const radius = _getDistance(startCanvas, endCanvas);
 
     // Checking if point is near the tool by comparing its distance from the center of the circle
-    return !(
-      distanceFromCenter > radius + distance / 2 || distanceFromCenter < radius
+    return (
+      distanceFromCenter > radius - distance / 2 &&
+      distanceFromCenter < radius + distance / 2
```

Comment:
This is great! Thank you.

Can we expand this out a bit farther? I've noticed that at the limits of this threshold, the annotation highlights green but is not draggable. There seems to be a slight disconnect between the values we're using for highlighting in the render loop and the values used here for `pointNearTool`

Reply:
@dannyrb : Thanks for the reply. 

I've noticed the same behavior with EllipticalTool. Sometimes, when annotation highlights, instead of dragging the annotation, mouse down will draw a new annotation. Any place you can advise to look into for the solution?

Reply:
![ellipticalToolThreshold](https://user-images.githubusercontent.com/11068601/57758977-3efcaa80-773c-11e9-9bfb-db7dc4546227.gif)